### DISCUSSION TITLE
Registered Models: more envs?

### DISCUSSION THREAD

**Discussion by harshilprajapati96** [mlflow/mlflow#6789] [Q&A] [answered]
Can we update the server to have more envs like dev or qa in addition to staging and production?

**Comment by JHibbard** [comment, marked as answer]
Hello, great question. As an alternative to additional stages, it's possible to create a convention within your group to use specific stage/model tag combinations to carry these additional intents. For example, you could have a model in staging that's tagged with "qa" or "dev". Hope that helps!
